app/code/Magento/Customer/etc/adminhtml/system.xml

```diff
@@ -86,6 +86,11 @@
                     <comment>To show VAT number on Storefront, set Show VAT Number on Storefront option to Yes.</comment>
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                 </field>
+                <field id="email_required_create_order" translate="label comment" type="select" sortOrder="58" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
```

Comment:
I think that this configuration is more relevant to the 'Magento_Sales' module.

Reply:
@swnsma 

Do we add the email address filed, Under General Section in Sales Module, or are you looking for somewhere else please suggest?

Please view the attachment for more details

![general_email_required](https://user-images.githubusercontent.com/16253886/64530780-9518dd80-d32b-11e9-8c24-34313be23e8b.png)

Reply:
@solwininfotech, I mean to move the changes into Magento/Sales/etc/adminhtml/system.xml.
As I can see, there location of config in Admin UI was already approved in https://github.com/magento/magento2/pull/22648

Reply:
@swnsma 

Yes Admin UI was already approved but that was under Customer Section but as per your revision, I need to add that in Sales Section.

In the Sales Section do I need to create a new Group with the name "Create New Order Options" and inside that, I give the option for Email is required field for Admin order creation?

Please advised.

Reply:
Hi @solwininfotech,
No, Customer Section - it is OK.
I am asking to move this changes to Magento Sales module.
Magento Sales has dependencies on Magento Customer and you can use Customer Section in that case to add you field from Magento Sales modules.
One more time: the section is Customer, but `system.xml` is placed in `Magento/Sales/etc/adminhtml/system.xml`

Reply:
@swnsma 

Done.

Please review it.